docs/running-on-yarn.md

```diff
@@ -134,6 +134,13 @@ Most of the configs are the same for Spark on YARN as for other deployment modes
   </td>
 </tr>
 <tr>
+  <td><code>spark.yarn.am.port</code></td>
+  <td>(random)</td>
+  <td>
+    Port for the YARN Application Master to listen on.
```

Comment:
Also kind of confusing is that in other places we use the `spark.yarn.am.*` namespace to refer only to the yarn-client application master.  If this applies to both yarn-client and yarn-cluster, the naming is kind of inconsistent.